modules/microshift-olm-deploy-ops-spec-ns.adoc

```diff
@@ -131,9 +131,9 @@ spec:
 +
 [source,terminal]
 ----
-$ oc apply -f _<my-catalog-source.yaml>_ <1>
+$ oc apply -f _<my_catalog_source>.yaml_ <1>
 ----
-<1> Replace _<my-catalog-source.yaml>_ with your catalog source configuration file name.
+<1> Replace `<my_catalog_source>` with your catalog source configuration file name.
```

Comment:
```suggestion
<1> Replace `_<my_catalog_source>_` with your catalog source configuration file name.
```

Reply:
@ShaunaDiaz  should I retain the original values "`_<my-catalog-source.yaml>_`"

Reply:
`_<my_catalog_source.yaml>_` is what we want in the footnote per the SSG
The code block should be _<my_catalog_source.yaml>_ (no backticks, but yes italics)

Reply:
done